Update concurrency category BBEs except strands BBE (#2906)
ballerina-platform/ballerina-distribution: examples/inter-worker-failure-propagation/inter_worker_failure_propagation.md

# Inter-worker failure propagation

> Workers may need to call functions that can return an `error`. Pairing up of sends and receives guarantees that each send will be received, and vice-versa, provided neither sending nor receiving worker has failed. Send to or receive from failed worker will propagate the failure.

Reviewer: ```suggestion
Workers may need to call functions that can return an `error`. Pairing up of sends and receives guarantees that each send will be received and vice-versa provided that the worker has failed on both sending and receiving. Sending or receiving of the failed worker will propagate the failure.
```
